requirements.txt

```diff
@@ -0,0 +1,9 @@
+asgiref==3.2.7
+dj-database-url==0.5.0
+Django==3.0.6
+django-heroku==0.3.1
+gunicorn==20.0.4
+psycopg2==2.8.5
+pytz==2020.1
+sqlparse==0.3.1
+whitenoise==5.1.0
```

Comment:
Actually the `Pipfile` was enough, I tried deleting the `requirements.txt` and it worked! :)
I misunderstood the Heroku documentation because it always referenced the `requirements.txt` file.